textract2page/cli.py

```diff
@@ -14,19 +14,26 @@
     type=click.Path(dir_okay=False, writable=True, exists=False, allow_dash=True),
 )
 @click.argument("aws-json-file", type=click.Path(dir_okay=False, exists=True))
-@click.argument("image-file", type=click.Path(dir_okay=False, exists=True))
-def cli(output_file, aws_json_file, image_file):
+@click.argument("image-file", type=str)
+@click.option("--image-width", type=int, help="Width of the image in pixels, if the image isn't available.")
+@click.option("--image-height", type=int, help="Height of the image in pixels, if the image isn't available.")
+def cli(output_file, aws_json_file, image_file, image_width, image_height):
     """Convert an AWS Textract JSON file to a PAGE XML file.
 
-    Also requires the original input image of AWS OCR to get absolute image coordinates.
+    Because of differences in the way Textract JSON and PAGE XML represent coordinates,
+    either the original image file must be supplied, or the image's filename and its 
+    absolute pixel dimensions must be supplied.
 
-    The output file will reference the image file under `Page/@imageFilename`
-    with its full path. (So you may want to use a relative path.)
+    The output file will reference the image file using the name you provide.
+    (So you may want to use a relative path.)
     """
```

Comment:
```suggestion
    """
    assert image_width and image_height or image_file and exists(image_file),\
        "requires passing either an existing image file path or --image-width and --image-height"
```

(and also at the top of the file: `from os.path import exists`)